### ISSUE TITLE
accessibility issue

### ISSUE THREAD

**Issue by earleyj1** [pkerspe/ckeditor-bootstrap-collapsibleItem#11]
There is an accessibility issue in that the chevron icon is NOT accessible. To make it accessible, the span with the icon should have
`<span aria-hidden="true" class="glyphicon glyphicon-chevron-down">&nbsp;</span><span class="sr-only">Expand/Collapse</span>`

However, it seems that the "sr-only" class is being stripped out, and I'm not sure why. CKEditor itself allows it, so I'm thinking its the plugin. 

I tried modifying 
` allowedContent: 'div(!collapsible-item*,panel*,collapse)[*];h4(!collapsible-item*,panel*)[*];a(!collapsible-item*,collapsed,panel*)[*];span(!glyphicon*)[*];',
`
by adding the class here, but it had no effect.
